### PR TITLE
Compare android version with semver

### PR DESCRIPTION
**Pull request by kosssi**

### CODE REVIEW

File: src/app/lib/database.coffee [hunk 1]
```diff
@@ -16,10 +17,10 @@ class Database
     #
     # To test:
     #   options = db: require 'memdown'
-    constructor: (options = adapter: 'idb', cache: false) ->
+    constructor: (options = adapter: 'idb', cache: false, device = device) ->
```
Comment (nono) [inline src/app/lib/database.coffee]: @kosssi I don't see from where the `device` is coming (in ` = device`)

Comment (kosssi) [inline src/app/lib/database.coffee]: it's cordova plugin ;)

Comment (kosssi) [inline src/app/lib/database.coffee]: it's cordova plugin ;)

Comment (nono) [inline src/app/lib/database.coffee]: Ok

**Pull request merged**